pkg/ddc/juicefs/transform_fuse.go

```diff
@@ -411,3 +424,75 @@ func (j *JuiceFSEngine) genFormatCmd(value *JuiceFS, config *[]string) {
 	cmd := append([]string{common.JuiceCliPath, "auth"}, args...)
 	value.Configs.FormatCmd = strings.Join(cmd, " ")
 }
+
+// getQuota: get quota from string
+func (j *JuiceFSEngine) getQuota(v string) (int64, error) {
+	q, err := resource.ParseQuantity(v)
+	if err != nil {
+		return 0, fmt.Errorf("invalid quota %s: %v", v, err)
+	}
+	qs := q.Value() / 1024 / 1024 / 1024
+	if qs <= 0 {
+		return 0, fmt.Errorf("quota %s is too small, at least 1GiB for quota", v)
+	}
+
+	return qs, nil
+}
+
+// genQuotaCmd: generate command for set quota of subpath
+func (j *JuiceFSEngine) genQuotaCmd(value *JuiceFS, mount datav1alpha1.Mount) error {
+	options := mount.Options
+	for k, v := range options {
+		if k == "quota" {
+			qs, err := j.getQuota(v)
+			if err != nil {
+				return err
```

Comment:
Suggest to wrap the err with `errors.Wrapf()` to describe the context of the error

Reply:
done